woocommerce/woocommerce-gateway-payfast
# Fix/46 Fix build script to include required directories into the distribution archive

Fixes: https://github.com/woocommerce/woocommerce-gateway-payfast/issues/46

The `composer archive` command is respecting `.gitignore`, that's why "build" directory was excluded from the zip. The same reason, `languages` directory was not included to the latest bundle.

Also, when in Sandbox mode and empty Merchant ID/Key/Passcode, Payfast method was globally disabled.

**How to test:**

1. Activate Payfast payment option (even Sandbox mode or live Merchant credentials)
2. Open the Checkout edit page with **Checkout block** (requires WooCommerce Blocks extension)
3. Payfast option expected to be visible in the **Payment section**

**Build script test:**

1. Run `npm run build`
2. The resulting archive expected to contain `assets`, `build`, `includes` and `languages` folders. Should not contain `.github`, `client` (source JS files), `vendor`, `node_modules` and developer-related config files in the root folder.
